docs/api-status.json

```diff
@@ -1963,6 +1963,12 @@
         "comment": "EncryptionLoad2 enables IO on an open encrypted image. Multiple encryption\noption values can be passed to this call in a slice. For more information\nabout how items in the slice are applied to images, and possibly ancestor\nimages refer to the documentation in the C api for rbd_encryption_load2.\n\nImplements:\n\n\tint rbd_encryption_load2(rbd_image_t image,\n\t                         const rbd_encryption_spec_t *specs,\n\t                         size_t spec_count);\n",
         "added_in_version": "v0.32.0",
         "expected_stable_version": "v0.34.0"
+      },
+      {
+        "name": "Image.DiffIterateByID",
+        "comment": "DiffIterateByID calls a callback on changed extents of an image.\n\nCalling DiffIterateByID will cause the callback specified in the\nDiffIterateConfig to be called as many times as there are changed\nregions in the image (controlled by the parameters as passed to librbd).\n\nSee the documentation of DiffIterateCallback for a description of the\narguments to the callback and the return behavior.\n\nImplements:\n\n\tint rbd_diff_iterate3(rbd_image_t image,\n\t                      uint64_t from_snap_id,\n\t                      uint64_t ofs, uint64_t len,\n\t                      uint8_t include_parent, uint8_t whole_object,\n\t                      int (*cb)(uint64_t, size_t, int, void *),\n\t                      void *arg);\n",
```

Comment:
- `DiffIterateConfig` -> `DiffIterateByIDConfig`
- `uint8_t include_parent, uint8_t whole_object` is no longer part of `rbd_diff_iterate3()` signature

Reply:
> @Rakshith-R as a reminder you should not need to edit this json directly. Edit the doc comment in the code and then you (after removing this hunk) can regenerate it using make like when you first added it.

I missed the removing part before running make api-updates.
It's updated now.